.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release from tags
+# Create a GitHub release when a tag is pushed.
+
+on:
+  push:
+    tags:
+      # this is a glob, not a regexp
+      - 'anaconda-*'
+
+permissions:
+  contents: write
+
+jobs:
+  release-from-tag:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Extract version from the tag
+        id: get_version
+        # If the version does not fit the regexp, we'll just fail. TODO maybe abort instead?
+        run: |
+          echo "Checking pushed tag ref ${{ github.ref }}"
+          VER=$(echo ${{ github.ref }} | perl -ne '/^refs\/tags\/anaconda-([0-9]+(?:\.[0-9]+){1,3})-1$/ && print "$1\n";')
+          if [ -z "$VER" ] ; then
+            echo "Tag ref ${{ github.ref }} is not a valid release tag."
+            exit 1
+          else
+            echo "Tag ref ${{ github.ref }} detected as release version $VER."
+            echo "::set-output name=version::$VER"
+          fi
+
+      - name: Check out repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+
+      #- name: Build anaconda-rpm container (for RPM build)
+      #  run: |
+      #    make -f Makefile.am anaconda-rpm-build
+
+      - name: Run the build in the container
+        run: |
+          mkdir /tmp/results
+          podman run \
+            -i \
+            -v /tmp/results:/results \
+            -v `pwd`:/anaconda \
+            quay.io/rhinstaller/anaconda-rpm:master <<EOF
+
+          # prepare environment
+          set -eux
+
+          # install dependencies for translation canary
+          dnf install -y git python3-polib python3-pip
+          pip install --upgrade pip pocketlint
+
+          # build tarball
+          cd /anaconda
+          ./autogen.sh
+          ./configure
+          make release
```

Comment:
I'm thinking if creating a new container wouldn't be really the best way to go. This container doesn't have to be build automatically and could be useful especially for RHEL releases or older Fedora where the dependencies on the system could be missing?

Reply:
A new container is a possibility. Same as the suggestion to use makefile, I dislike that it would scatter the actual steps all around the place. Right now this single workflow file is the only place, with everything needed. If that is of no value to you, let's try exploding it to a new container (based on the rpm container) + script in the container + makefile target.

Reply:
I thought that these steps could be reusable in other cases (for local use), but I might be wrong. Would it be possible to create a scratch build?

Reply:
There is already `scripts/testing/install_dependencies.sh` to set up a machine and then you just run the three commands above. That's the documented way to do things. Should we wrap it up in another makefile target? It might be possible. But that's out of scope here.

A scratch build does not have the translations which means it will not produce the desired "upstream release".

Reply:
> There is already `scripts/testing/install_dependencies.sh` to set up a machine and then you just run the three commands above. That's the documented way to do things. Should we wrap it up in another makefile target? It might be possible. But that's out of scope here.

That doesn't have to work with RHEL branches.

> A scratch build does not have the translations which means it will not produce the desired "upstream release".

Yes, you need to call a different `make` command, but there are ways how to change the default call.

It is just an idea. I don't insist on it.

Reply:
...we will make upstream RHEL releases?